paloesg/excide
# Enhance/add documents to tasks motif

# Description

Add functionality to add documents to tasks from the edit drawer in the edit template page.

Notion link: https://www.notion.so/0b9be2495a23438995cc85fd0680affa?v=b8425151e89b4b14bfe425fc33d558b5&p=615b407f276e4e1aa03575fda20452f4

## Remarks

- Removed the table to add documents upon task creation (_new_drawer.html.slim) since the workflow action has yet to be instantiated, so any documents uploaded cannot belong to it.

# Testing

- Add new tasks in a motif template
- After creation edit the task and click 'Upload' to add a file to it
- Check that in the edit drawer, the file uploaded is shown
